docs/en/reference/information_schema/tasks.md

```diff
@@ -0,0 +1,18 @@
+---
+displayed_sidebar: "English"
+---
+
+# tasks
+
+`tasks` provides information about asynchronous tasks.
+
+The following fields are provided in `tasks`:
+
+| **Field**   | **Description**                                              |
+| ----------- | ------------------------------------------------------------ |
+| TASK_NAME   | Name of the task.                                            |
+| CREATE_TIME | Time when the task is created.                               |
+| SCHEDULE    | Task schedule. If the task is regularly triggered, this field displays `START xxx EVERY xxx`. |
+| DATABASE    | Database where the task belongs.                             |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/d2190e9f-c045-4f2c-b67d-226f370e4630)
-> to which